src/commands/server/start.ts

```diff
@@ -142,6 +142,13 @@ export default class Start extends Command {
       this.exit()
     }
 
+    if (flags.platform === 'minikube' && flags.installer && flags.installer.includes('minishift')) {
+      platformCheckTasks.add({
+        title: '🛑  Minishift Requirement',
+        task: () => this.error(`Current platform is ${flags.platform }. Minishift installers are only available on top of Minishift platform.`)
```

Comment:
Why are you talking about "minishift installers" instead of "minishift-addon installer"? I mean I am not sure how users will interpret "minishift installers": can the operator considered a minishift installer?

Reply:
because I checked for 'minishift' string being included in the installer in case other installers would include minishift in their name

Reply:
but I can be dumb and check for `minishift-addon` directly

Reply:
Well we support 3 installers today and one of those is called `minishift-addon`. I don't know if we will ever have other `*minishift*` installers.

Another thing you could add in this PR is a check that `helm` installer is NOT used with the minishift platform. What do you think?